docs/vars.md

```diff
@@ -16,7 +16,7 @@ aliases:
 updated at Mon May  8 06:43:29 UTC 2023
 
 
-| varible name | variable default value | variable required | variable description |
```

Comment:
This typo will appear again after generation.
The source of typo is here: https://github.com/f41gh7/envconfig-docs/blob/8e8f96cc6387e74970082e80395a56191538ef33/main.go#L55 (it's a tool by @f41gh7 used for generation of this doc).